maintenance/scripts/refresh_api.sh

```diff
@@ -1,9 +1,9 @@
 #! /bin/bash
 
-parent_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+parent_path=~/gc-digital-talent/maintenance/scripts
```

Comment:
Was this change on purpose or just testing? It breaks the script for me locally and seems out of scope for this issue.

Reply:
Hmm, I thought I changed this back with this commit 😕  https://github.com/GCTC-NTGC/gc-digital-talent/pull/5840/commits/0c2dc4d57518c4b3366e7f52fa8118143a2c0ac5. Here is the fix again [19071bf](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5840/commits/19071bf51947d7318eaf34077b8499a0d84260de)